tools/src/main/java/org/apache/kafka/tools/MetadataQuorumCommand.java

```diff
@@ -96,14 +100,18 @@ static void execute(String... args) throws Exception {
                 if (namespace.getBoolean("status") && namespace.getBoolean("replication")) {
                     throw new TerseException("Only one of --status or --replication should be specified with describe sub-command");
                 } else if (namespace.getBoolean("replication")) {
-                    handleDescribeReplication(admin);
+                    boolean humanReadable = Optional.of(namespace.getBoolean("human_readable")).orElse(false);
+                    handleDescribeReplication(admin, humanReadable);
                 } else if (namespace.getBoolean("status")) {
+                    if (namespace.getBoolean("human_readable")) {
+                        throw new TerseException("The option -hr/--human-readable is only supported along with --replication");
```

Comment:
Since we don't have `-hr` option, this message should be updated.

Reply:
Thanks for catching this.